CMake/cdat_modules_extra/cdat_common_environment.cmake.in

```diff
@@ -1,5 +1,5 @@
 message("[INFO] ADDITIONAL CFLAGS ${ADDITIONAL_CFLAGS}")
-set(ENV{PATH} "@cdat_EXTERNALS@/bin:$ENV{PATH}")
+set(ENV{PATH} "@SB_BIN_DIR@:@cdat_EXTERNALS@/bin:$ENV{PATH}")
```

Comment:
This is the one in `cdat_common_environment`, where it isn't strictly necessary (and when I tried removing it from scipy's build with it in here, I ran into the same cython issue).